presto-hive-metastore/src/main/java/com/facebook/presto/hive/TypeTranslator.java

```diff
@@ -16,7 +16,11 @@
 import com.facebook.presto.common.type.Type;
 import org.apache.hadoop.hive.serde2.typeinfo.TypeInfo;
 
+import java.util.Optional;
+
 public interface TypeTranslator
 {
     TypeInfo translate(Type type);
```

Comment:
Let's set a default implementation here that calls  `translate(type, Optional.empty())`